src/main/resources/META-INF/rewrite/java-version-11.yml

```diff
@@ -73,6 +73,11 @@ recipeList:
       getPeerMethodPattern: java.awt.* getPeer()
       lightweightPeerFQCN: java.awt.peer.LightweightPeer
   - org.openrewrite.scala.migrate.UpgradeScala_2_12
+  - org.openrewrite.java.migrate.RemovedLogManagerPropertyChangeListenerMethods
```

Comment:
Let's remove this line that carried over from
- https://github.com/openrewrite/rewrite-migrate-java/pull/526

Reply:
```suggestion
```